sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/CurrentUser.scala

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
Add this to https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/misc.scala?